### ISSUE TITLE
Adding GLEU score 

### ISSUE THREAD

**Issue by alvations** [google/seq2seq#34]
`nltk` has a simple implementation of GLEU score as described in Wu et al.'s (2016) GMNT system: https://github.com/nltk/nltk/blob/develop/nltk/translate/gleu_score.py 

It'll be great if the `seq2seq` has a similar port of GLEU =)

**Comment by dennybritz** [comment]
Yup, would be a good addition to BLEU and ROUGE. I don't want to make NLTK a dependency, but it's simple enough to just add from scratch.

**Comment by alvations** [comment]
Is there a particular place (directory) to put metrics and significance computation scripts?

**Comment by dennybritz** [comment]
Yeah, in `[metrics/]`(https://github.com/google/seq2seq/tree/master/seq2seq/metrics). After adding the metric as a python function there you also need to add a `MetricSpec` that can be used with Tensorflow, [`like this`](https://github.com/google/seq2seq/blob/master/seq2seq/metrics/metric_specs.py#L149)
